pandas/core/series.py

```diff
@@ -2221,26 +2220,36 @@ def combine(self, other, func, fill_value=np.nan):
         0    0
         1    2
         dtype: int64
-
         See Also
         --------
         Series.combine_first : Combine Series values, choosing the calling
             Series's values first
         """
+        self_is_ext = is_extension_array_dtype(self.values)
```

Comment:
I would move all of this (combine stuff) to another PR on top of this one.

Reply:
https://github.com/pandas-dev/pandas/pull/21183